README.md

```diff
@@ -92,3 +92,39 @@ explicitly or implicitly down-cased.
 
 For example, the Http header `X-B3-ParentSpanId: 0020000000000001` would become
 an ASCII header `x-b3-parentspanid` with the same value.
+
+# Frequently Asked Questions
+
+## Why is ParentSpanId propagated?
+
+In B3, the trace context is extracted from headers sent upstream. Timing and
```

Comment:
Maybe
`In B3, the trace context is extracted from headers (the SpanID)` 
I really do not know what `sent upstream` means